include/stout/flags/flags.cc

```diff
@@ -587,18 +569,39 @@ void Parser::Parse(const std::vector<ArgumentInfo>& values) {
   }
 
   // Ensure required flags are present.
-  for (const auto& [_, field] : fields_) {
+  for (const auto& [name, field] : fields_) {
+    // The field has been parsed, all good, just continue.
+    if (parsed_.count(field) > 0) {
+      continue;
+    }
     const stout::v1::Flag& flag =
         field->options().GetExtension(stout::v1::flag);
 
-    // This field is not required, so just continue.
-    if (!flag.required())
+    // Need to normalize string default value before parsing.
+    // NOTE: for some reason protoc compiler has generated
+    // 'default_()' method with underscore, we believe that
+    // this happens because 'default' conflicts with reserved
+    // words.
+    const std::string normalized_value =
+        GetNormalizedDefaultValue(
+            flag.default_(),
+            field->type() == google::protobuf::FieldDescriptor::TYPE_STRING);
```

Comment:
Maybe pass in the value of `field->type()` into the helper along with the `== TYPE_STRING` comparison to avoid repetition